iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/CommentLocationVisitor.java

```diff
@@ -44,7 +44,8 @@
 public class CommentLocationVisitor extends TreeVisitor<InputFileContext> {
   private static final Logger LOG = LoggerFactory.getLogger(CommentLocationVisitor.class);
 
-  private static final Pattern CONTAINS_LINE_NUMBER = Pattern.compile("#(?<number>\\d+)(\\s#\\d+)*+$");
+  private static final Pattern CONTAINS_LINE_NUMBER = Pattern.compile("#(?<number>\\d++)(\\s#\\d++:?\\d*+)*+$");
+  private static final Pattern CONTAINS_LINE_NUMBER_RANGE = Pattern.compile("#(?<rangeStart>\\d++):(?<rangeEnd>\\d++)(\\s#\\d++:?\\d*+)*+$", Pattern.UNICODE_CHARACTER_CLASS);
```

Comment:
I guess we can use ` ` (simple space) instead of `\\s` as this is in our control and we always use space. Maybe then we can omit `Pattern.UNICODE_CHARACTER_CLASS`?

Reply:
I'm not sure what is the logic behind the rule that suggested adding this. I think `\p*` are the only character classes that were present in the rule's description.